src/Types/ConfirmType.res

```diff
@@ -1,6 +1,7 @@
 type confirmParams = {
   return_url: string,
   publishableKey: string,
+  redirect?: string,
```

Comment:
There is no need to have redirect as an optional param

Reply:
It's required else unnecessary I have to pass if_required everywhere

Reply:
It's not required for wallets